### PR TITLE
1.3 "rename for X11"

### PR DESCRIPTION
**Pull request by zvezdochiot**
Rename utilites for X11.

$ ls /usr/bin/x*

/usr/bin/x11copy
...
/usr/bin/xsct
...
/usr/bin/xzoom

### CODE REVIEW

File: changelog [hunk 1]
```diff
@@ -0,0 +1,17 @@
+PROJECT: https://github.com/faf0/sct
+
+1.3: zvezdochiot on 02 Jan 2018
+* Rename utilites for X11
+* Add -h option to print usage information
+* Add license documentation
+* Add changelog
```
Comment (faf0) [inline changelog]: please rename this file to `CHANGELOG` ( no need to update this line)

File: sct.c [hunk 2]
```diff
@@ -18,94 +18,108 @@
...
 #include <X11/Xatom.h>
 #include <X11/Xlib.h>
 #include <X11/Xproto.h>
 #include <X11/extensions/Xrandr.h>
 
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+
+void
```
Comment (faf0) [inline sct.c]:
```suggestion
static void
```

File: xsct.1 [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+
+.SH OPTIONS
+.IP temperature
+If passed a value in the correct range (see above)
+.B xsct
+will set the current screen temperature to this value
+.IP -h,--help
+Display usage information and exit
+.IP none
+If no options are passed xsct sets the color temperature to the default of 6500
```
Comment (faf0) [inline xsct.1]: comma after `passed`

**Comment by faf0** [comment]
Thanks @zvezdochiot !

**Comment by zvezdochiot** [comment]
Ok. I agree.

File: changelog (hunk 1 above)
Comment (zvezdochiot) [reply changelog]: I agree.

File: sct.c (hunk 2 above)
Comment (zvezdochiot) [reply sct.c]: I agree.

File: xsct.1 (hunk 3 above)
Comment (zvezdochiot) [reply xsct.1]: Right

**Review by faf0 — APPROVED** [review]
Thanks for this pull request!

**Pull request merged**